docs/docs/reference/smart_contract_reference/macros.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Aztec macros
+sidebar_position: 6
+---
+
+## All Aztec macros
+
+In addition to the function macros in Noir, Aztec also has its own macros for specific functions. An Aztec contract function can be annotated with more than 1 macro.
+It is also worth mentioning Noir's `unconstrained` function type [here](https://noir-lang.org/docs/noir/concepts/unconstrained/).
+
+- `#[aztec(public)]` or `#[aztec(private)]` - Whether the function is public or private (more in next section)
```

Comment:
Deleted doc split into inner workings and this reference, so is no longer "next section".

```suggestion
- `#[aztec(public)]` or `#[aztec(private)]` - Whether the function is to be executed from a public or private context (see Further Reading)
```